src/commands/local-store/local-store-command.js

```diff
@@ -75,6 +81,110 @@ class LocalStoreCommand extends Command {
                     );
                 }
                 await Promise.all(storePromises);
+            } else if (storeType === LOCAL_STORE_TYPES.TRIPLE_PARANET) {
+                const paranetMetadata = await this.blockchainModuleManager.getParanetMetadata(
+                    blockchain,
+                    paranetId,
+                );
+                const paranetUAL = this.ualService.deriveUAL(
+                    blockchain,
+                    paranetMetadata.paranetKAStorageContract,
+                    paranetMetadata.paranetKATokenId,
+                );
+                const paranetRepository = this.paranetService.getParanetRepositoryName(paranetUAL);
+
+                if (cachedData.public.assertion && cachedData.public.assertionId) {
+                    let attempts = 0;
+                    while (attempts < LOCAL_INSERT_MAX_ATTEMPTS) {
+                        try {
+                            // eslint-disable-next-line no-await-in-loop
+                            await this.tripleStoreService.localStoreAsset(
+                                paranetRepository,
+                                cachedData.public.assertionId,
+                                cachedData.public.assertion,
+                                blockchain,
+                                contract,
+                                tokenId,
+                                keyword,
+                            );
+                            break;
+                        } catch (error) {
+                            attempts += 1;
+                            if (attempts < LOCAL_INSERT_MAX_ATTEMPTS) {
+                                // eslint-disable-next-line no-await-in-loop
+                                await setTimeout(LOCAL_INSERT_RETRY_DELAY);
+                            } else {
+                                // eslint-disable-next-line no-await-in-loop
+                                await this.tripleStoreService.deleteAssetMetadata(
+                                    paranetRepository,
+                                    blockchain,
+                                    contract,
+                                    tokenId,
+                                );
+                                // eslint-disable-next-line no-await-in-loop
+                                await this.tripleStoreService.deleteAssertion(
```

Comment:
Problem with that is if insertion failed, we may delete assertion, but it can actually be the same assertion of other KA (same applies to private assertion)

Reply:
Maybe just delete metdata?